sbnobj/Common/Reco/MVAPID.cc

```diff
@@ -0,0 +1,25 @@
+#include "sbnobj/Common/Reco/MVAPID.h"
+
+#include "cetlib_except/exception.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+
+#include <algorithm>
+
+void sbn::MVAPID::AddScore(int pdg, float score)
+{
+  auto [iter, success] = mMVAScoreMap.try_emplace(pdg, score);
```

Comment:
I didn't know you could do multiple assignment like this in C++ yet, cool.

That said, you don't need the iter so I guess you could write `if(!mMVAScoreMap.try_emplace(pdg, score).second){`